### ISSUE TITLE
Incorrect preview in mardown editor for html blockcode

### ISSUE THREAD

**Issue by SharkProgramming** [Requarks/wiki#1179]
**Describe the bug**
Incorrect preview in mardown editor for html blockcode or inline code

**To Reproduce**
1. Go to New page
2. Choose Markdown Editor
3. Write HTML in inline code or block code with language html defined

**Expected behavior**
We have just to see the html like presented in the blockcode not the interpretation of the html

**Screenshots**
![issue](https://user-images.githubusercontent.com/33226400/68189604-f536a380-ffab-11e9-9ba6-df9ee05f2928.png)

**Error Message**
La ressource à l’adresse « http://192.168.8.86:1337/e/en/style.css » a été bloquée en raison d’un type MIME (« text/html ») incorrect (X-Content-Type-Options: nosniff).

**Host Info (please complete the following information):**
 - OS: Ubuntu 18.04.3
 - Wiki.js version: 2.0.0-rc.17
 - Database engine: MariaDB 10.3.18

**Comment by dash00** [comment]
I have a similar example and actually I think this issue could lead to a security issue. 
![image](https://user-images.gituusercontent.com/6683784/69788263-6ab61e00-11be-11ea-89b0-9ec1fd0d25d5.png)

Some observations:

- If I save the page the final rendering is fine. 

- If I remove the `<` and `>` characters, the preview in the editor is fine. 

- If I play with the editor and I use `<h1>` here is what happens:
![image](https://user-images.githubusercontent.com/6683784/69788384-abae3280-11be-11ea-92d8-c97c861dc6e7.png)

Host info:
- OS: Docker Container (Linux)
- Wiki.js: 2.0.12
- Database engine: PostgreSQL 9.6.15

**Comment by changsijay** [comment]
I found similar issue when toggling preview pane, which make the code block become blur and shadowed font.

**Comment by ccolella-mdc** [comment]
I am having this issue as well.

**Comment by Yivan** [comment]
Same here, I just wanted to add linked task: https://github.com/Requarks/wiki/issues/1370

**Comment by Yivan** [comment]
@NGPixel I would like to add that it should be **really prioritized please**, because it can break totally the preview page, meaning we can no more preview at all.

Using this code:
```php
<?php

include __DIR__ . "/../../../vendor/autoload.php"; // chemin à adapter selon l'emplacement du script

\Pimcore\Bootstrap::setProjectRoot();
\Pimcore\Bootstrap::bootstrap();

// On peut maintenant appeler les méthodes Pimcore

// On peut aussi paramétrer PHP
error_reporting(E_ALL); // on affiche même les erreurs de niveau notice
set_time_limit(0); // pas de limite de temps
ini_set('memory_limit', '2048M'); // on alloue davantage de mémoire
```

Lead to:

![image](https://user-images.githubusercontent.com/9025839/72282914-f9331180-363d-11ea-8b8a-f69f9ede0d27.png)

I hope it could really be fixed soon. As it is a core feature the preview mode and very helpfull.

Thanks a lot.

**Comment by RichardD2** [comment]
I'm seeing something similar in v2.1.113, trying to add Razor code to a Markdown page. The preview shows the code perfectly, but the page itself doesn't.

Markdown:
![image](https://user-images.githubusercontent.com/2158265/75561886-9eb01200-5a3f-11ea-86ae-87c4c6af588c.png)

Preview:
![image](https://user-images.githubusercontent.com/2158265/75561912-ab346a80-5a3f-11ea-8aa0-01a3ec0625f7.png)

Rendered page:
![image](https://user-images.githubusercontent.com/2158265/75561943-b8515980-5a3f-11ea-84e7-004f745e7467.png)

**Comment by RichardD2** [comment]
If I HTML-encode the HTML block in the Markdown, the preview shows the encoded value, but the page shows the correct value.

Markdown:
![image](https://user-images.githubusercontent.com/2158265/75562813-1fbbd900-5a41-11ea-93ac-8554d65db252.png)

Preview:
![image](https://user-images.githubusercontent.com/2158265/75562844-28acaa80-5a41-11ea-85e8-e1a640b4ebb7.png)

Page:
![image](https://user-images.githubusercontent.com/2158265/75562861-319d7c00-5a41-11ea-986f-6e9018aed034.png)

**Comment by NGPixel** [comment]
Fixed by b529ad21c97a7093e9ecc35e2e72fb8148f0c9d0
